src/test/java/net/minecraftforge/debug/FluidMixTest.java

```diff
@@ -0,0 +1,25 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.block.BlockStone;
+import net.minecraft.init.Blocks;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.BiomeHills;
+import net.minecraftforge.event.world.BlockEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid = "fluidmixtest", name = "FluidMixTest", version = "0.0.0")
+@Mod.EventBusSubscriber @SuppressWarnings("unused")
+public class FluidMixTest
+{
```

Comment:
This test mod should be disableable and disabled by default, so that it doesn't interfere with testing of other test mods or creation of other PRs.